tools/cli-client/internals/trigger/build_metadata.go

```diff
@@ -49,12 +49,16 @@ func InferBuildMetadata() BuildMetadata {
 	// use regex to match the repo location
 	regex := regexp.MustCompile("github.com[:/](canonical/.*).git")
```

Comment:
We should only match "Push URL: ..." (even without ".git"), because there can be the case the the developer pulls from upstream and updates the repo in canonical. (Suggested by @clay-lake )

Reply:
Updated.